### DISCUSSION TITLE
Why am I being redirected to the wrong url when using auth redirectTo option?

### DISCUSSION THREAD

**Discussion by GaryAustin1** [supabase/supabase#12946] [Troubleshooting]
Also, Why am I getting redirected to localhost instead of my production site URL?

In order for the provided redirectTo option to work you must set the exact URL in the shown Redirect URL's setting.

![image](https://user-images.githubusercontent.com/54564956/224379580-fa77bd31-bb58-47e6-90ce-64e140f32579.png)

For more information on formats for redirect URL settings see the documentation here: https://supabase.com/docs/guides/auth/redirect-urls#use-wildcards-in-redirect-urls  (edited to new link)

**Comment by aantti** [comment]
Has the documentation section moved here? https://supabase.com/docs/guides/auth/redirect-urls

**Reply by GaryAustin1** [reply]
Appears so:  https://supabase.com/docs/guides/auth/redirect-urls#use-wildcards-in-redirect-urls
